tests/bundle/SystemInfo/Collector/JsonComposerLockSystemInfoCollectorTest.php

```diff
@@ -40,7 +40,7 @@ public function testCollectWithMinimumStability(): void
                     'license' => 'GPL-2.0',
                     'stability' => 'dev',
                     'dateTime' => new \DateTime('2016-02-28 14:30:53'),
-                    'homepage' => 'http://share.ez.no',
+                    'homepage' => 'http://share.ibexa.co',
```

Comment:
This URL is incorrect
```suggestion
                    'homepage' => 'https://ibexa.co',
```